### PR TITLE
fix: use unified memo field and  extract memo value from different coin transaction implementation/labels

### PR DESCRIPTION
**Pull request by themooneer**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description
- Handle memo tag values extracted from different transaction properties
- Unify memo tag field for SOL, TON, STELLAR 

https://github.com/user-attachments/assets/9cd8ee64-92b8-488f-a419-28a4eea4c4d1

https://github.com/user-attachments/assets/816b45e4-970c-4873-bd0d-ceb07c73dc64

https://github.com/user-attachments/assets/7a35935b-5d81-4a81-90e9-3447dcd4a4b2

https://github.com/user-attachments/assets/f1fa17e0-3a78-449b-9080-47cdde56ec8c

https://github.com/user-attachments/assets/810b725d-2a38-49c9-9115-a6561ad7ffcb

https://github.com/user-attachments/assets/b992eacc-ec81-4d93-afcb-3f669dd775c2

https://github.com/user-attachments/assets/95883f91-79b4-41aa-a7cb-1c3199d9d8c9

### ❓ Context

- **JIRA or GitHub link**: 

https://ledgerhq.atlassian.net/browse/LIVE-14902
https://ledgerhq.atlassian.net/browse/LIVE-14910

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/modals/Send/steps/StepSummary.tsx [hunk 1]
```diff
@@ -84,8 +84,13 @@ const StepSummary = (props: StepProps) => {
   const specific = currency ? getLLDCoinFamily(mainAccount.currency.family) : null;
   const SpecificSummaryNetworkFeesRow = specific?.StepSummaryNetworkFeesRow;
 
-  const memo = "memo" in transaction ? transaction.memo : undefined;
-
+  const memo = lldMemoTag?.enabled
```
Comment (KVNLS) [inline apps/ledger-live-desktop/src/renderer/modals/Send/steps/StepSummary.tsx:87]: Can we please create a dedicated function to retrieve the memo/tag?
Can we use something like transaction.family to know which coin we are creating a tx for, please?

File: apps/ledger-live-desktop/src/renderer/modals/Send/steps/StepRecipient.tsx [hunk 2]
```diff
@@ -153,9 +153,16 @@ export const StepRecipientFooter = ({
   const alwaysShowMemoTagInfo = useSelector(alwaysShowMemoTagInfoSelector);
 
   const handleOnNext = async () => {
+    const memoTagValue =
```
Comment (KVNLS) [inline apps/ledger-live-desktop/src/renderer/modals/Send/steps/StepRecipient.tsx:156]: Same here

**Review by KVNLS — CHANGES_REQUESTED** [review]
When I do not select anything for XLM in the memo/tag dropdown, the prompt about "No memo/tag" does not appear.
https://github.com/user-attachments/assets/86e678b9-820e-42c1-93df-ab421e432766

**Comment by themooneer** [comment]
@KVNLS i've fixed this https://github.com/LedgerHQ/ledger-live/pull/8392#pullrequestreview-2444699596 review , it's an existing functional bug that in my opinion should be fixed without feature flag because it has an impact on the old and new memoTagFlow

File: .changeset/early-pugs-refuse.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"ledger-live-desktop": patch
```
Comment (qperrot) [inline .changeset/early-pugs-refuse.md:2]: Now we need to use `minor` and not `patch` anymore

Comment (themooneer) [reply .changeset/early-pugs-refuse.md]: done ☑️

**Review by qperrot — APPROVED** [review]
Need to rebase to fix CLI

**Review by KVNLS — APPROVED** [review]
LGTM!
A more general feedback, when I check the "do not show again" option and I reload the app, the prompt still appears and can you fix on another PR the margin issue we have between the two CTAs, please?
image

**Pull request merged**